libraries/testing-ui/src/main/kotlin/com/bumble/appyx/testing/ui/rules/AppyxViewTestRule.kt

```diff
@@ -47,7 +48,7 @@ open class AppyxViewTestRule<View : NodeView>(
     override fun beforeActivityLaunched() {
         AppyxTestActivity.composableView = {
             CompositionLocalProvider(
-                LocalNode provides DummyParentNode<Any>(),
+                LocalNode provides DummyParentNode<Any>().build(),
```

Comment:
I am fine keeping this as the only change of PR.

Reply:
ok. let's keep only this change otherwise it gets ugly to make this check only for view testing use case